src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -81,18 +81,18 @@ public static Address parseAddress(String address) throws ParseException {
     }
 
     /**
-     * Parses a {@code String email} into an {@code Email}.
+     * Parses a {@code String expiry date} into an {@code ExpiryDate}.
```

Comment:
Should "String expiry date" be "String expiryDate" to match the argument?

Reply:
Noted. Thanks.